packages/eslint-config-mc-app/package.json

```diff
@@ -22,8 +22,8 @@
   "module": "./index.js",
   "peerDependencies": {
     "babel-eslint": "10.x",
-    "eslint": "5.x || 6.x",
-    "eslint-config-airbnb-base": "13.x",
+    "eslint": "^5.16.0 || ^6.1.0",
```

Comment:
This is the same range as airbnb has. Stricly speaking it is a breaking change cause we go from requireing 5.x to 5.16. I assume airbnb has reasons for 5.16 but I am not sure. @emmenko WDYT?